File: app/src/main/java/com/example/hashisushi/utils/CustomFonts.java
package com.example.hashisushi.utils;

import android.graphics.Typeface;
import android.support.v7.app.AppCompatActivity;

public class CustomFonts extends AppCompatActivity {
	
	//Altera fonte do txtLogo
	public Typeface fontLogo ( ) {
		Typeface font = Typeface.createFromAsset ( getAssets ( ), "RagingRedLotusBB.ttf" );
		return font;
	}
}

File: app/src/main/java/com/example/hashisushi/utils/CustomFonts.kt
package com.example.hashisushi.utils

import android.graphics.Typeface
import android.support.v7.app.AppCompatActivity

class CustomFonts : AppCompatActivity() {

    //Altera fonte do txtLogo
    fun fontLogo(): Typeface {
        return Typeface.createFromAsset(assets, "RagingRedLotusBB.ttf")
    }
}
